docs/faq.rst

```diff
@@ -155,7 +155,7 @@ Why are there no wheels for my Python3.x version?
 Our Python3 wheels are ``abi3`` wheels. This means they support multiple
 versions of Python. The ``abi3`` wheel can be used with any version of Python
 greater than or equal to the version it specifies. Recent versions of ``pip``
-will automatically install ``abi3`` wheels.
+(20.0.1+) will automatically install ``abi3`` wheels.
```

Comment:
Why 20.0.1? There's nothing in the changelog that obviously indicates that's when abi3 was added https://pip.pypa.io/en/stable/news/#v20-0-1

Reply:
This is around when pip added abi3 for windows, but other platforms had support far earlier. Trying to specify an exact version here won't be simple because the required pip version is platform and cryptography version dependent. This is generally why all of our installation error messages suggest upgrading pip to latest.

Reply:
Indeed I also looked at the changelog in vain - so sadly had to resort to binary search. I discovered that `19.3.1` doesn't handle abi3 correctly and `20.0.1` does. Per https://pypi.org/project/pip/#history `20.0` was yanked and there are no other released versions of pip inbetween.
![image](https://user-images.githubusercontent.com/793763/147072558-31f300b1-296f-4bd3-ba26-42450dbedd44.png)

It must have been fixed somewhere in the 16kloc diff between https://github.com/pypa/pip/compare/19.3.1...20.0.1 but I didn't have the time or motivation to pin it down further I'm afraid (!)

Reply:
20.0.1 probably is correct, but that's strictly for Windows. Since this is different for every platform and it is our general advice that users upgrade to the latest `pip` I'm not sure what better language is available than `recent` for this.

Reply:
Fair enough, I didn't realise pip gained this support for different platforms at different times so I figured stating an exact version would be non-controversial and might help some.

"recent" will of course get less true the longer it has been since 20.0.1 but I can't think of a better non-specific word without listing a version number so if you'd rather not do so that's your call of course :)